src/version.h

```diff
@@ -20,7 +20,7 @@ static const int INIT_PROTO_VERSION = 209;
 static const int MIN_PEER_PROTO_VERSION = 70216;
 
 //! minimum proto version of masternode to accept in DKGs
-static const int MIN_MASTERNODE_PROTO_VERSION = 70235;
+static const int MIN_MASTERNODE_PROTO_VERSION = 70237;
```

Comment:
it should not bump `MIN_MASTERNODE_PROTO_VERSION` if that's a *patch* version IMO. It means all masternodes *must* update, right?